doc/Changes.md

```diff
@@ -0,0 +1,6344 @@
+    accidentally changed. This change may have side-effects in users'
+    code, for example the object returned by `0 * ComputedPowerMaps(
+    CharacterTable( "A5" ) )[2]` had been a mutable list before the
+    change, and is an immutable list from now on.
+
+##### 2.1-8 Removed or obsolete functionality
+
+  - Remove multiple undocumented internal functions. Nobody should have
+    been using them, but if you were, you may extract it from a previous
+    **GAP** release that still contained it.
+    ([\#2670](https://github.com/gap-system/gap/pull/2670),
+    [\#2781](https://github.com/gap-system/gap/pull/2781) and more)
+
+  - [\#2335](https://github.com/gap-system/gap/pull/2335) Remove several
+    functions and variables that were deprecated for a long time:
+    `DiagonalizeIntMatNormDriven`, `DeclarePackageDocumentation`,
+    `KERNEL_VERSION`, `GAP_ROOT_PATHS`, `LOADED_PACKAGES`,
+    `PACKAGES_VERSIONS`, `IsTuple`, `StateRandom`, `RestoreStateRandom`,
+    `StatusRandom`, `FactorCosetOperation`, `ShrinkCoeffs`,
+    `ExcludeFromAutoload`, `CharacterTableDisplayPrintLegendDefault`,
+    `ConnectGroupAndCharacterTable`, `IsSemilatticeAsSemigroup`,
+    `CreateCompletionFiles`, `PositionFirstComponent`, `ViewLength`
+
+  - [\#2502](https://github.com/gap-system/gap/pull/2502) Various kernel
+    functions now validate their inputs more carefully (making it harder
+    to produce bad effects by accidentally passing bad data to them)
+
+  - [\#2700](https://github.com/gap-system/gap/pull/2700) Forbid
+    constructors with 0 arguments (they were never meaningful)
+
+##### 2.1-9 Packages
+
+**GAP** 4.10.0 distribution includes 140 packages.
+
+Added to the distribution:
+
+  - The **francy** package by Manuel Martins, which provides an
+    interface to draw graphics using objects. This interface allows
+    creation of directed and undirected graphs, trees, line charts, bar
+    charts and scatter charts. These graphical objects are drawn inside
+    a canvas that includes a space for menus and to display informative
+    messages. Within the canvas it is possible to interact with the
+    graphical objects by clicking, selecting, dragging and zooming.
+
+  - The **JupyterVis** package by Nathan Carter, which is intended for
+    use in Jupyter Notebooks running **GAP** kernels and adds
+    visualization tools for use in such notebooks, such as charts and
+    graphs.
+
+No longer redistributed with **GAP**:
+
+  - The **linboxing** package has been unusable (it does not compile)
+    for several years now, and is unmaintained. It was therefore dropped
+    from the **GAP** package distribution. If anybody is willing to take
+    over and fix the package, the latest sources are available at
+    <https://github.com/gap-packages/linboxing>.
+
+  - The **recogbase** package has been merged into the `recog` package,
+    and therefore is no longer distributed with
+**GAP**.
+
+#### 2.2 **GAP** 4.10.1 (February 2019)
```

Comment:
So right now the order of releases in this file is:
- 4.11.0
- 4.10.0
- 4.10.1
- 4.10.2
- 4.9.0
- 4.9.1
- ...

That's not good. We should permute it.